com.unity.ml-agents/Runtime/Actuators/ActuatorList.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Unity.MLAgents.Actuators
+{
+    public class ActuatorList : IList<IActuator>
```

Comment:
Not sure if this is worth the code, but I didn't like this buffer management code living in `Agent`

Reply:
This part sounds good, might just want to make the class itself internal, and maybe make clear what's required for the IList interface vs what's actually useful (e.g. don't think we'll be calling IndexOf much, but we have to implement it).

Reply:
Yeah, I'll add "decide on class visibility" as part of my TODO list.